lamindb_setup/core/upath.py

```diff
@@ -550,7 +550,9 @@ def get_stat_dir_gs(path: UPath) -> Tuple[int, str, str, int]:
     return sum(sizes), hash, hash_type, n_objects
 
 
-def check_s3_storage_location_empty(path: UPath) -> None:
+def check_s3_storage_location_empty(path: UPathStr) -> None:
+    if not isinstance(path, UPath):
+        path = convert_pathlike(path)
     objects = path.fs.find(path.as_posix())
     n_objects = len(objects)
     if n_objects > 1:
```

Comment:
i think it might be better to use size instead of n_objects. touched file has 0 size i think.

Reply:
So we don't really need to take the changing number of objects into account. btw i i haven't checked but what happens for sqlite 33 instances? They always have at least an sqlite file in the bucket.

Reply:
Good point. We're going to have a problem with sqlite instances if we condition instance deletion on size being 0.

I think for this case we're necessarily going to have to check for the presence of the specific instance sqlite file based on its naming convention, no?

Reply:
Yes

Reply:
We also need to account for the `_lamindb_initialized` file, don't we?

Reply:
What is `_lamindb_initialized`? Is it always written to `.lamindb/` in the storage location?